__tests__/unit/lib/utils/repoGitDiff.test.js

```diff
@@ -257,18 +258,110 @@ describe(`test if repoGitDiff`, () => {
       // eslint-disable-next-line no-undef
       globalMetadata
     )
-    const work = repoGitDiff.getDiff()
+    const work = repoGitDiff.getFilteredDiff()
     expect(work).toStrictEqual(output)
   })
 
+  test('can explicitly include files', () => {
```

Comment:
**praises:** that is awesome ! unit test for all the positive test cases

Reply:
👍